File: Review/Python/review-5.py
class Time:
    def __init__(self, hour, minute, second):
        self.hour = hour
        self.minute = minute
        self.second = second

def main():
    time = []

    for _ in range(2):
        hour, minute, second = map(int, input().split())
        time.append(Time(hour, minute, second))
    
    total1 = time[0].hour * 3600 + time[0].minute * 60 + time[0].second
    total2 = time[1].hour * 3600 + time[1].minute * 60 + time[1].second

    diff = abs(total2 - total1)
    print(diff // 3600, (diff // 60) % 60, diff % 60)

if __name__ == '__main__':
    main()

File: Review/C/review-5.c
#pragma warning(disable:4996)
#include <stdio.h>

typedef struct Time {
    int hour;
    int minute;
    int second;
}Time;

int main() {

    Time time[2];

    for(int i = 0; i < 2; i++)
        scanf("%d %d %d", &time[i].hour, &time[i].minute, &time[i].second);
    
    int total1 = time[0].hour * 3600 + time[0].minute * 60 + time[0].second;
    int total2 = time[1].hour * 3600 + time[1].minute * 60 + time[1].second;

    int diff;
    if(total1 < total2)
        diff = total2 - total1;
    else
        diff = total1 - total2;

    printf("%d %d %d", diff / 3600, diff / 60 % 60, diff % 60);

    return 0;
}